### PR TITLE
add sample app

### PR DESCRIPTION
**Pull request by tomkukral**

### CODE REVIEW

File: app/kad/server.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+from .middleware import setup_metrics
+
+app = Flask(__name__)
+redis_server = os.environ.get('REDIS_SERVER', 'localhost')
+config_file = os.environ.get('CONFIG_FILE', '/etc/kad/config.yml')
+redis = Redis(host=redis_server, port=6379)
+notready_file = '/tmp/notready'
+
+
+app = Flask(__name__)
```
Comment (jwerak) [inline app/kad/server.py:26]: Why duplicate?

File: app/setup.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
+"""Setup for kqueen package."""
```
Comment (jwerak) [inline app/setup.py]: what is this kqueen, did you even work on that? Check the license...

**Review by jwerak — APPROVED** [review]

File: app/kad/server.py (hunk 1 above)
Comment (tomkukral) [reply app/kad/server.py:26]: By mistake, good catch!

**Review by jwerak — APPROVED** [review]
LGTM

**Pull request merged**